tests/console/install_otherDE_pattern.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    my $self = shift;
+    select_console 'root-console';
+
+    script_run("zypper lr -d | tee /dev/$serialdev");
+
+    my $pattern   = get_var("DE_PATTERN");
+    my $zypp_type = "pattern";
+    if (check_var("DE_IS_PKG", 1)) {
```

Comment:
Just as DE_PATTERN comes from the test suite, that's the point when we know if we want to install a pattern or actually only a package that pulls in more... defining them together makes most sense imho